eng/pipelines/templates/steps/analyze.yml

```diff
@@ -3,16 +3,11 @@ parameters:
   ServiceDirectory: ''
   TestMarkArgument: ''
   AdditionalTestArgs: ''
+  Artifacts: []
 
 steps:
   - template: /eng/pipelines/templates/steps/analyze_dependency.yml
 
-  - task: PythonScript@0
```

Comment:
Why is this being removed?

Reply:
We don't need to use that again since there is a common changelog verification.

Reply:
Oh I see it below :)